### PR TITLE
Toasts

### PR DESCRIPTION
**Pull request by AnnaShchelokova**
## Type of Change (Select One and Apply Github Label)

- [ ] Bug fix (non-breaking change which fixes an issue) -> Apply bugfix label
- [ ] New feature (non-breaking change which adds functionality) -> Apply enhancement label
- [ ] Breaking change (fix or feature that would cause existing functionality to change) -> Apply breaking label

## Which browsers have you tested?
- [ ] Internet Explorer 11
- [ ] Edge
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

## Checklist:

- [ ] I have read the (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md] document.
- [ ] My code passes the automated linter.
- [ ] This code has been reviewed by another team member and passes the reviewer checklist found in (https://github.comv/GSA/sam-ui-elements/blob/CONTRIBUTING.md)[CONTRIBUTING.md]
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] My code is 508 compliant as tested by AMP and JAWS
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/sam-design-system-site/src/app/app.module.ts [hunk 1]
```diff
@@ -11,7 +11,8 @@ import { library } from '@fortawesome/fontawesome-svg-core';
 import { sds } from '@gsa-sam/sam-styles/src/icons/';
 import { fas } from '@fortawesome/free-solid-svg-icons';
 import { MarkdownModule } from 'ngx-markdown';
-
+import { Toast, ToastrModule } from 'ngx-toastr';
+import {SdsToastComponent} from 'libs/packages/components/src/lib/toast/toast-single.component';
```
Comment (yerramshilpa) [inline apps/sam-design-system-site/src/app/app.module.ts:15]: Can you change the reference to use the package name?

File: apps/sam-design-system-site/src/app/app.module.ts [hunk 2]
```diff
@@ -26,14 +27,20 @@ export function hljsLanguages() {
 }
 
 @NgModule({
-  declarations: [AppComponent],
+  declarations: [AppComponent,SdsToastComponent],
```
Comment (yerramshilpa) [inline apps/sam-design-system-site/src/app/app.module.ts:30]: Is there any reason why we need to add a component in the app-level..?

File: libs/packages/sam-material-extensions/src/lib/accordion/accordion.component.spec.ts [hunk 3]
```diff
@@ -74,7 +74,7 @@ describe('SdsAccordionComponent', () => {
       expect(component).toBeTruthy();
     }));
 
-    it('should have 4 items', () => {
+    it('should have sd items', () => {
       expect(component.accordionItems.length).toEqual(4);
```
Comment (yerramshilpa) [inline libs/packages/sam-material-extensions/src/lib/accordion/accordion.component.spec.ts:78]: This change is not required. Can you revert changes

File: package.json [hunk 4]
```diff
@@ -95,6 +95,7 @@
     "@types/jasmine": "~2.8.6",
     "@types/jasminewd2": "~2.0.3",
     "@types/jest": "^23.0.0",
+    "ngx-toastr": "^10.2.0",
```
Comment (yerramshilpa) [inline package.json:98]: Can you also add the package name to the peer dependencies of the package library?

File: apps/sam-design-system-site/src/app/app.module.ts (hunk 2 above)
Comment (AnnaShchelokova) [reply apps/sam-design-system-site/src/app/app.module.ts:30]: refuses to work any other way

**Review by davereed — APPROVED** [review]

**Review by davereed — APPROVED** [review]

**Pull request merged**